src/pygenesis/helloworld.py

```diff
@@ -0,0 +1,49 @@
+"""Module provides wrapper for HelloWorld GENESIS REST-API functions."""
+
+import requests
+
+from pygenesis.config import load_config
+from pygenesis.http_helper import _check_invalid_status_code
+
+
+def whoami() -> str:
+    """
+    Wrapper method which constructs an URL for testing the Destatis API
+    whoami method, which returns host name and IP address.
+
+    Returns:
+        str: text test response from Destatis
+    """
+    config = load_config()
+    url = f"{config['GENESIS API']['base_url']}" + "helloworld/whoami"
+
+    response = requests.get(url, timeout=(1, 15))
+
+    _check_invalid_status_code(response.status_code)
+
+    return str(response.text)
```

Comment:
str cast sollte hier nicht notwendig sein, laut requests doku immer unicode: Content of the response, in unicode.

Reply:
Hab mich hierbei an den aktuellen http_helpern orientiert, ist jetzt dann aber auch gefixt

Reply:
Hab es wieder zurückgenommen, da mypy sonst "Any" als type für response.text annimmt